internal/server/errors_test.go

```diff
@@ -46,17 +46,6 @@ func TestSendAPIError(t *testing.T) {
 			err:    fmt.Errorf("hide this: %w", internal.ErrForbidden),
 			result: api.Error{Code: http.StatusForbidden, Message: "forbidden"},
 		},
-		{
-			err:    internal.ErrDuplicate,
-			result: api.Error{Code: http.StatusConflict, Message: "duplicate record"},
-		},
```

Comment:
We should replace these test cases  with ones that use `data.UniqueConstraintError`, to show it uses the right status code.